pkg/registry/apis/provisioning/resources/repository.go

```diff
@@ -10,9 +10,13 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
+type RepositoryResourcesOptions struct {
+	CommitWithOriginalAuthors bool
```

Comment:
IIUC, this is really "lookup all users at startup" -- ideally we could lookup a single user if we are only doing a single thing.

Though we may be able to just get it from the context 🤔

Reply:
PreloadAllUserInfo?